modules/phase_field/src/auxkernels/TestEBSDAux.C

```diff
@@ -24,6 +24,11 @@ TestEBSDAux::TestEBSDAux(const InputParameters & parameters) :
 {
 }
 
+TestEBSDAux::~TestEBSDAux()
+{
+  delete _val;
```

Comment:
Yes, it's a "build" type method that probably does need to allocate memory in order to work correctly, we just need to manage its lifetime better.